charts/aad-pod-identity/templates/mic-deployment.yaml

```diff
@@ -62,6 +62,9 @@ spec:
           {{- if .Values.mic.clientQps }}
           - --clientQps={{ .Values.mic.clientQps }}
           {{- end }}
+          {{- if .Values.mic.immutableUserMSIs }}
+          - --immutableUserMSIs={{ .Values.mic.immutableUserMSIs }}
```

Comment:
consider passing the IDs as a list and concatenate them here with `join ","`

Reply:
Done